src/pages/Inventory.page.js

```diff
@@ -1,15 +1,68 @@
 const { BaseSwagLabPage } = require('./BaseSwagLab.page');
 
+export const SORTING_OPTIONS_FUNCTIONS = {
+    az: (a, b) => a.name.localeCompare(b.name),
+    za: (a, b) => b.name.localeCompare(a.name),
+    lohi: (a, b) => a.price - b.price,
+    hilo: (a, b) => b.price - a.price,
+};
+
 export class InventoryPage extends BaseSwagLabPage {
     url = '/inventory.html';
 
-    get headerTitle() { return this.page.locator('.title'); } //
+    get headerTitle() {
+        return this.page.locator('.title');
+    }
 
-    get inventoryItems() { return this.page.locator('.inventory_item'); }
+    get sortingSelect() {
+        return this.page.locator('.product_sort_container');
+    }
 
-    get addItemToCartBtns() { return this.page.locator('[id^="add-to-cart"]'); }
+    get activeOption() {
+        return this.page.locator('.active_option').textContent();
+    }
+
+    get inventoryItems() {
+        return this.page.locator('.inventory_item');
+    }
 
     async addItemToCartById(id) {
-        await this.addItemToCartBtns.nth(id).click();
+        const item = this.inventoryItems.nth(id);
+        await item.getByRole('button', { name: 'Add to cart' }).click();
+    }
+
+    async sortingBy(value) {
+        await this.sortingSelect.selectOption(value);
+    }
+
+    async getSortedItems(option = null) {
```

Comment:
I'd recommend having a method that returns the data from the page without making any changes to the data. 
And perform sort right in the test, but not in the Page Object